lib/astc_codec.cpp

```diff
@@ -1080,7 +1291,7 @@ ktxTexture2_DecodeAstc(ktxTexture2 *This, ktx_uint32_t vkformat) {
     astcenc_profile profile{ASTCENC_PRF_LDR_SRGB};
```

Comment:
Is this necessary? You are assigning a new value to `profile` at line 1089. If it is necessary, shouldn't you be only be using SRGB if the incoming texture is sRGB_

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/967/commits/b7acc0aa7d4b45c8286a341dae73bdd8101fe21c